scss/bitstyles/settings/color-palette.stories.mdx

```diff
@@ -0,0 +1,152 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Design Tokens/Color palette" />
+
+# Color palette
+
+The global color palette is used throughout the components and utility classes of bitstyles to maintain visual consistency.
+
+By default the palette includes variants of the following colors:
+
+```scss
+$black: #000;
+$white: #fff;
+$brand-1: #000dff;
+$brand-2: #fa0;
+$text: #333;
+$background: color.mix($white, $brand-1, 97%);
+$positive: #50a802;
+$danger: #e82915;
+$warning: #fe881e;
+```
+
+For each of these colors we generate two maps of 18 color variants: one has white mixed into the color (made available with the name of the color) and one has black mixed in (available with the name of the color suffixed with `_shades`). There is also a map of `gray` colors, which by default is your brand-1 color mixed with black through to white.
```

Comment:
Why 18 by the way? It's such a non-nerdy number 🤷

Reply:
It is! 😂 Something I only realised when I wrote this. I’ll add a note to maybe update that (to 20?) in #618, thanks